libazureinit/src/provision/ssh.rs

```diff
@@ -1,5 +1,10 @@
 // Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
+// Licensed under the MIT License.A
```

Comment:
Super small typo with an additional `A` here. Everything else looks great! Thanks for all the work you're doing.

Reply:
Didn't notice that, thanks! Done :)